src/crystal_lib/parser.cr

```diff
@@ -15,6 +15,14 @@ class CrystalLib::Parser
     @nodes = [] of ASTNode
     @cursor_hash_to_node = {} of UInt32 => ASTNode
     @idx = Clang::Index.new
+
+    # include clang builtin headers when necessary
+    # (see https://clang.llvm.org/docs/FAQ.html#i-get-errors-about-some-headers-being-missing-stddef-h-stdarg-h)
+    clanglibdir = "/usr/include/clang/#{`llvm-config --version`.strip}/include/"
```

Comment:
This may not be necessary (and/or may conflict) if `brew link` command is done above.  `brew link` should put the stuff in /usr/local/include/clang, and based on what it says in https://clang.llvm.org/docs/FAQ.html#i-get-errors-about-some-headers-being-missing-stddef-h-stdarg-h , I think it will then find the headers.

Reply:
You're right, this fix is not necessary for macOS machines but it is for -at least- Debian/Ubuntu Linux installs.

Should I add a `flag?(:darwin)` check ?

Reply:
I think that [this](http://clang-developers.42468.n3.nabble.com/Default-header-search-paths-not-included-when-compiling-from-libclang-td4036193.html) is the problem we are experiencing here.

Still, I'm not really sure what would be the better way to fix it.